### PR TITLE
Contact form implimentation

### PR DESCRIPTION
**Pull request by RWUBAKWANAYO**
# In this project we: 

- Add contact section in HTML file
- Add styles to created section
- Invite learning partner to contribute to this project
- setup mobile version of new section
- Fix linter errors in the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -220,6 +222,18 @@ <h3>Skills</h3>
...
+  <section class="contact-container">
+    <h2>Contact me</h2>
+    <p>
+      If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+    </p>
+    <form class="contact-form" action="https://formspree.io/f/xayvbzrj" method="post">
+      <input type="text" placeholder="Enter your name" name="name" maxlength="30" required />
+      <input type="email" placeholder="Enter your email" name="email" required />
+      <textarea rows="10" maxlength="500" placeholder="Write your message here" name="message" required ></textarea>
+      <input type="submit" value="Get It Touch" id="contact-submit" />
```
Comment (brytebee) [inline index.html:234]: - [ ] **Required:** You have done an amazing job 👏 . However, it is required that your button is interactive.

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi Team 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- amazing design✔️
- no linter errors✔️
- no console errors✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] **Optional:** Kindly update the live demo link on your readme. It is broken.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mateo951) [reply index.html:234]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @RWUBAKWANAYO, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💪😎 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**